apsuite/commissioning_scripts/calc_orbcorr_mat.py

```diff
@@ -176,6 +175,13 @@ def __init__(self, model, acc):
         self.ch_idx = self._get_idx(self.ch_idx)
         self.cv_idx = self._get_idx(self.fam_data['CV']['index'])
 
+        if self.nturns > 1 and acc in ('BO', 'SI'):
+            self.model = self.nturns*model
+            self.bpm_idx = pyaccel.lattice.find_indices(
+                self.model, 'fam_name', 'BPM')
+        else:
+            self.bpm_idx = self._get_idx(self.fam_data['BPM']['index'])
+
```

Comment:
@murilobalves, I think these changes were in the other PR as well, weren't they?

Reply:
ops, I messed up, I'll redo the branch and PR.